sdk/monitor/Azure.Monitor.Query/tests/MonitorQueryTestEnvironment.cs

```diff
@@ -28,7 +28,6 @@ public class MonitorQueryTestEnvironment : TestEnvironment
         public string ConnectionString => GetRecordedOptionalVariable("CONNECTION_STRING");
         public string StorageAccountId => GetRecordedOptionalVariable("STORAGE_ID");
         public string StorageAccountConnectionString => GetRecordedOptionalVariable("STORAGE_CONNECTION_STRING");
-        public string MetricsLocation => GetRecordedVariable("METRICS_LOCATION");
```

Comment:
Does this need to be deleted from the bicep?

Reply:
I think it is deleted :)